src/System.Private.ServiceModel/tests/Common/Infrastructure/testproperties.props

```diff
@@ -150,6 +150,10 @@
     <PropertyGroup Condition="'$(SSL_Available)' == ''">
       <SSL_Available></SSL_Available>
     </PropertyGroup>
+
+  <PropertyGroup Condition="$(WithCategories.Contains('failing'))">
```

Comment:
Just trying to understand... when `$(WithCategories.Contains('failing'))` will be true?